pkg/model/proto/model.proto

```diff
@@ -1,664 +1,667 @@
-// This source file has been modified by Huawei Technologies Co., Ltd.
```

Comment:
It seems that the format of this file is not right.

Reply:
done